src/openrct2-ui/interface/ViewportInteraction.cpp

```diff
@@ -363,7 +363,8 @@ InteractionInfo ViewportInteractionGetItemRight(const ScreenCoordsXY& screenCoor
             }
 
             ride->FormatNameTo(ft);
-            ft.Add<rct_string_id>(RideComponentNames[RideTypeDescriptors[ride->type].NameConvention.station].capitalised);
+            ft.Add<rct_string_id>(
+                RideComponentNames[EnumValue(RideTypeDescriptors[ride->type].NameConvention.station)].capitalised);
```

Comment:
Make an accessor function for this as it is used in tons of places.

`constexpr const RideComponentName& GetRideComponentName(const RideComponentType type){ return RideComponentNames[EnumValue(type)];}`